main.sh

```diff
@@ -459,6 +484,7 @@ for tool in "${tools[@]}"; do
     info "${tool} installed at $(type -P "${bin}")"
     case "${bin}" in
         "cargo-udeps${exe}") x cargo udeps --help | head -1 ;; # cargo-udeps v0.1.30 does not support --version option
+        "cargo-valgrind${exe}") x cargo valgrind --help ;;     # cargo-valgrind v2.1.0 does not support --version option
```

Comment:
The [help text](https://github.com/jfrimmel/cargo-valgrind/blob/43b31c9304a7c0eb3d867711ead98951e8d91da1/src/main.rs#L62-L74) isn't too long so I've opted to not pipe to `head` to avoid `failed printing to stdout: The pipe is being closed` panic on Windows.